### ISSUE TITLE
Dedicated social media page 

### ISSUE THREAD

**Issue by Pradhvan** [pythonindia/inpycon2020#51]
I recently saw the EuroPython's social media[ page](https://ep2020.europython.eu/social-media/) I think it would be good to have feature for PyCon India too since it can be used by us(social media team) to point to a single place which has all the info and also would be helpful for attendees before/ during the conference.

**Comment by anistark** [comment]
Do we have the content/links ready for this?

cc: @Pradhvan @vipulgupta2048 @Sc0rpi0n101

**Comment by vipulgupta2048** [comment]
I thought this might not be critical enough for the website team, since we are running low on volunteers. So we included all the handles on the first question of FAQ section (for now) All the links are there.

**Comment by anistark** [comment]
If this is not required anymore given FAQ is done, can you close the issue? @vipulgupta2048 

Otherwise, I'll work on it once other issue that currently working on is done.

**Comment by vipulgupta2048** [comment]
@anistark I would suggest to keep it open if someone else wants to take it up later.

**Comment by anistark** [comment]
If all the necessary handles are already included in FAQ and this is not required, please close it.

If it's still required, please share the links and content and I'll work on it and push it. Let's not delay these any longer.

**Comment by Pradhvan** [comment]
@anistark It's not a priority but let's not close this as during and after the conference it would be easier to point people to the social media page instead of the FAQ. 

> I thought this might not be critical enough for the website team, since we are running low on volunteers.

as @vipulgupta2048 said website team is a bit low on volunteers so will it be possible if we can take this maybe later, some days prior to the conference maybe. When the website team isn't that much busy.

**Comment by Pradhvan** [comment]
Also if we can follow the same design template as the FAQ page(seeing that won't require much of front-end work) I can pitch to work on the issue.

**Comment by vipulgupta2048** [comment]
@Pradhvan Assign it to you then? You can take it up when you wish. I agree on the part where there's not a lot of frontend work involved in building more pages.
